frontend/tet/admin/src/components/editor/companyInfo/CompanyInfo.tsx

```diff
@@ -78,7 +84,7 @@ const CompanyInfo: React.FC = () => {
               options={keywords}
               optionLabelField={'label'}
               filter={addressFilterHandler}
-              validation={{ required: { value: true, message: t('common:editor.posting.validation.required') } }}
```

Comment:
Hmm, I think that the react-hook-form basic validation should work? Maybe the object was in wrong format?
I have had it like this for example: ` validation={{ required: true, maxLength: 64 }}`

Reply:
Problem was that posting's location key's value is set with initial posting data from PreviewContext.tsx. Since location key is not optional, whole Location type object is set (with empty strings and arrays). This seen as a value in validation, since it is not just 'null' and error for the field is not shown, if form is saved without changes on this field.